ckanext/archiver/plugin/__init__.py

```diff
@@ -69,7 +69,7 @@ def _is_it_sufficient_change_to_run_archiver(self, package, operation):
         # Since 'revisions' is a deprecated feature in CKAN,
         # try to use activity stream to check if dataset changed
         context = {'model': model, 'session': model.Session, 'ignore_auth': True, 'user': None}
-        if p.toolkit.check_ckan_version(min_version='2.8.0'):
+        if p.toolkit.check_ckan_version(min_version='2.9.0'):
```

Comment:
so package_activity_list is not in 2.8?

Reply:
It probably is, but since we can still use revisions on 2.8, it doesn't seem necessary to rewrite this code.